CHANGELOG.md

```diff
@@ -1,15 +1,29 @@
 # CHANGELOG
 
-## Unreleased
+## v3.2.0
 ### Breaking changes
+[\#550](https://github.com/Manta-Network/Manta/pull/550) Remove sudo pallet from calamari runtime.
```

Comment:
is this a breaking change? What exactly does it break for a user?

Reply:
It's important, because we're going to remove sudo from calamari.
Let me move it to `Improvements`.

Reply:
Sounds good. The "Breaking" section is really not about importance of the change, but whether someone who uses the node (e.g. frontend, polkadot.js) has to change their code or some functionality they used to have will no longer work after this node upgrade.